tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -640,3 +641,24 @@ def change_service_state(self, service_name, operation):
         self.ceph_rbdmirror.exec_command(
             sudo=True, cmd=f"systemctl {operation} {service_name}"
         )
+
+    def resize_image(self, imagespec, size):
+        """
+        Resize given provided image
+        Args:
+            imagespec: image-spec of the image to be resized
+            size: size of the image to be updated to
+        Returns:
+            None
+        Note:
+            Function raises IOonSecondaryError if resize is tried on
+        secondary image
+        """
+        log.info(f"Resizing image {imagespec} to size {size}")
+        cmd = f"rbd resize {imagespec} -s {size} --allow-shrink"
```

Comment:
Can we please have some IO put into image then get the free space and shrink.

Reply:
IO is being initiated from tests/rbd_mirror/test_9500_shrink_img_at_secondary.py
(This function is just to call resize)